SourceX/storm/storm.cpp

```diff
@@ -425,7 +425,7 @@ static bool HaveAudio()
 
 void SVidRestartMixer()
 {
-	if (Mix_OpenAudio(22050, AUDIO_S16SYS, 2, 1024) < 0) {
+	if (Mix_OpenAudio(22050, AUDIO_S16LSB, 2, 1024) < 0) {

```

Comment:
This fixes some sounds sounding wrong, like the two menu sounds (both in-game and in UI).

Reply:
Odd, I'm suprised it works at all on Big Endian..